apps/src/lib/levelbuilder/lesson-editor/SearchBox.jsx

```diff
@@ -12,13 +12,6 @@ export default class SearchBox extends Component {
     additionalQueryParams: PropTypes.object
   };
 
-  constructor(props) {
-    super(props);
-    this.state = {
-      searchValue: ''
```

Comment:
`searchValue` wasn't ever being updated so I removed it. When I did that, turns out this whole constructor is unnecessary.